.changeset/2565.md

```diff
@@ -0,0 +1,59 @@
+---
+'@backstage/core-api': minor
```

Comment:
Should actually be `patch` right?

Reply:
Done in https://github.com/spotify/backstage/pull/2724/commits/ee1b98030c35cf9a45788342ca89e5197e6f01e6